azurerm/resource_arm_sql_database.go

```diff
@@ -57,6 +58,56 @@ func resourceArmSqlDatabase() *schema.Resource {
 				DiffSuppressFunc: ignoreCaseDiffSuppressFunc,
 			},
 
+			"import": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"storage_uri": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"storage_key": {
+							Type:      schema.TypeString,
+							Required:  true,
+							Sensitive: true,
+						},
+						"storage_key_type": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"StorageAccessKey", "SharedAccessKey",
+							}, true),
```

Comment:
this `true` represents the strings being case-insensitive; as such can we also add `DiffSuppressFunc: ignoreCaseDiffSuppressFunc` here to handle the casing being different?